scripts/serve-local.sh

```diff
@@ -1,10 +1,12 @@
 #!/usr/bin/env bash
+echo "starting..."
 
 set -o errexit
 
 TAG='janis:local'
-
+echo "building docker image..."
```

Comment:
I originally added this echo for debugging and just left it as I found after multiple docker stops and starts, docker build would hang for lengthy periods even after container/image/layer cache clean up. So this will just let a dev know something is happening.